share/rocm/cmake/ROCMConfig.cmake

```diff
@@ -1,6 +1,9 @@
 # ######################################################################################################################
-# Copyright (C) 2017 Advanced Micro Devices, Inc.
+# Copyright (C) 2023 Advanced Micro Devices, Inc.
 # ######################################################################################################################
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
-include(ROCMChecks)
+message(INFO
+    "Use of find_package(ROCM) is deprecated as of <version>, and will be removed in <version>, "
+    "please switch to find_package(ROCmCMakeBuildTools)."
+)
+find_package(ROCmCMakeBuildTools PATHS .)
```

Comment:
My understanding is that the `PATHS` property is for searching static paths. If want to search a path that is relative to some other package, the appropriate property is `HINTS`. I'd also be more comfortable searching `"${CMAKE_CURRENT_LIST_DIR}"` rather than `.`.

Reply:
Agreed on both counts, updated.